packages/airnode-deployer/src/infrastructure/index.test.ts

```diff
@@ -18,6 +18,11 @@ jest.mock('../../package.json', () => ({
   version: '0.8.0',
 }));
 
+jest.mock('../utils/logger', () => ({
+  ...jest.requireActual('../utils/logger'),
+  writeLog: jest.fn(),
+}));
+
```

Comment:
I tried this mock, but for some reason it resulted in logger having this structure:

{
  ...(regular logger functions),
  default: {
    writeLog: mock
      ...(regular logger functions),
  }
}

so `writeLog` was never actually mocked correctly. Mocking `fs.appendFileSync` as in the last commit works, though.